src/change-listeners.js

```diff
@@ -198,6 +200,13 @@ class ChangeListeners extends Process {
     }
   }
 
+  async destroy(dbNames) {
```

Comment:
Is this destroy function being used?

Reply:
No.  That's just cruft from an initial, ill-advised attempt.  I'll clean it out.